COPYING

```diff
@@ -39,245 +39,71 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
-  OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
-  WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
-  ARE DISCLAIMED. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY
-  DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-  DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE
-  GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
-  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
-  WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
-  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
-  SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+See autogen.sh.  See licenses/autogen.sh-LICENSE for copyright and licensing
+information.
 
 CVC4 incorporates the script "doxygen.am", excluded from the above copyright.
-See config/doxygen.am.  Its copyright:
-
-  Copyright (C) 2004 Oren Ben-Kiki
-  This file is distributed under the same terms as the Automake macro files.
+See config/doxygen.am.  See licenses/doxygen.am-LICENSE for copyright and
+licensing information.
 
 CVC4 incorporates the m4 macro file "pkg.m4", excluded from the above
-copyright.  See config/pkg.m4.  Its copyright:
-
-  Copyright (c) 2004 Scott James Remnant <scott@netsplit.com>.
-
-  This program is free software; you can redistribute it and/or modify
-  it under the terms of the GNU General Public License as published by
-  the Free Software Foundation; either version 2 of the License, or
-  (at your option) any later version.
-
-  This program is distributed in the hope that it will be useful, but
-  WITHOUT ANY WARRANTY; without even the implied warranty of
-  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
-  General Public License for more details.
-
-  You should have received a copy of the GNU General Public License
-  along with this program; if not, write to the Free Software
-  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
-
-  As a special exception to the GNU General Public License, if you
-  distribute this file as part of a program that contains a
-  configuration script generated by Autoconf, you may include it under
-  the same distribution terms that you use for the rest of that program.
+copyright.  See licenses/pkg.m4-LICENSE for copyright and licensing
+information.
 
 CVC4 incorporates the m4 macro file "boost.m4", excluded from the above
 copyright and downloaded from http://github.com/tsuna/boost.m4 .
-See config/boost.m4.  Its copyright:
-
-  Copyright (C) 2007  Benoit Sigoure <tsuna@lrde.epita.fr>
-
-  This program is free software: you can redistribute it and/or modify
-  it under the terms of the GNU General Public License as published by
-  the Free Software Foundation, either version 3 of the License, or
-  (at your option) any later version.
+See config/boost.m4.  See licenses/boost.m4-LICENSE for copyright and licensing
```

Comment:
The "See config/boost.m4" is inconsistent with the other paragraphs, they do not have such a sentence.